api4/plugin.go

```diff
@@ -103,52 +104,39 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	downloadUrl := r.URL.Query().Get("plugin_download_url")
-
-	if !model.IsValidHttpUrl(downloadUrl) {
-		c.Err = model.NewAppError("installPluginFromUrl", "api.plugin.install.invalid_url.app_error", nil, "", http.StatusBadRequest)
-		return
+	force := false
+	if r.URL.Query().Get("force") == "true" {
```

Comment:
i know @iomodo this was already there but wondering if we can clean it up by just doing 
`force  := r.URL.Query().Get("force") == "true"`